spec/02-kong_storage_adapter_spec.lua

```diff
@@ -217,8 +236,10 @@ for _, strategy in helpers.each_strategy() do
 
         assert.response(res).has.status(200)
 
+        local found = db.sessions:select_by_session_id(sid)
```

Comment:
Is the fact that the status code is 200 good enough, or should we also assert that there was no error executing this query? `found` can be nil for two reasons: 1) it's not in the db, and 2) the select query bombed out.